packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -234,13 +234,15 @@ export class Choice extends React.PureComponent<
             }}
             {...inputProps}
           />
-          <FormLabel
-            className={labelClassName}
-            fieldId={this.id}
-            {...{ errorMessage, errorMessageClassName, hint, inversed, requirementLabel }}
-          >
-            {label}
-          </FormLabel>
+          <span>
```

Comment:
I had to wrap this label in a span to solve a layout issue. 

Because the previous iteration of label wrapped all the label-adjacent elements (label, hint, error), it played nicely with the flex/grid display being used on choice elements. But when this was refactored to pull hint and error text our of the label, it jacked up the layout. All elements need to be wrapped in something to lay out correctly.